LICENSE.build

```diff
@@ -1,19 +0,0 @@
-Copyright (c) 2019 The Meson development team
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
Why are you removing this? Are these license terms not suitable for you? The review policy requires this file to exist.

Reply:
Looking at existing repos, I only saw this file in master